.gitignore

```diff
@@ -27,6 +27,7 @@ target/
 
 # Generated during runtime
 /shmem
+/failed_snapshot_tests
```

Comment:
It would be fine to copy those files only when running on CI, so this would not be needed, but if it's already implemented I'm fine with leaving it for non ci envs too.

Reply:
I found it convenient to have failed test snapshots when writing tests locally